Formula/acmetool.rb

```diff
@@ -15,167 +14,27 @@ class Acmetool < Formula
     sha256 "62ec2c87880494488a50d78c36104f75eb97bb160ddf316387ab116e51ace2fd" => :high_sierra
   end
 
-  depends_on "go" => :build
-
-  uses_from_macos "libpcap"
-
-  go_resource "github.com/alecthomas/template" do
-    url "https://github.com/alecthomas/template.git",
-        :revision => "a0175ee3bccc567396460bf5acd36800cb10c49c"
-  end
-
-  go_resource "github.com/alecthomas/units" do
-    url "https://github.com/alecthomas/units.git",
-        :revision => "2efee857e7cfd4f3d0138cc3cbb1b4966962b93a"
-  end
-
-  go_resource "github.com/coreos/go-systemd" do
-    url "https://github.com/coreos/go-systemd.git",
-        :revision => "cc4f39464dc797b91c8025330de585294c2a6950"
-  end
-
-  go_resource "github.com/hlandau/buildinfo" do
-    url "https://github.com/hlandau/buildinfo.git",
-        :revision => "337a29b5499734e584d4630ce535af64c5fe7813"
-  end
-
-  go_resource "github.com/hlandau/dexlogconfig" do
-    url "https://github.com/hlandau/dexlogconfig.git",
-        :revision => "244f29bd260884993b176cd14ef2f7631f6f3c18"
-  end
-
-  go_resource "github.com/hlandau/goutils" do
-    url "https://github.com/hlandau/goutils.git",
-        :revision => "0cdb66aea5b843822af6fdffc21286b8fe8379c4"
-  end
-
-  go_resource "github.com/hlandau/xlog" do
-    url "https://github.com/hlandau/xlog.git",
-        :revision => "197ef798aed28e08ed3e176e678fda81be993a31"
-  end
-
-  go_resource "github.com/jmhodges/clock" do
-    url "https://github.com/jmhodges/clock.git",
-        :revision => "880ee4c335489bc78d01e4d0a254ae880734bc15"
-  end
-
-  go_resource "github.com/mattn/go-isatty" do
-    url "https://github.com/mattn/go-isatty.git",
-        :revision => "6ca4dbf54d38eea1a992b3c722a76a5d1c4cb25c"
-  end
-
-  go_resource "github.com/mattn/go-runewidth" do
-    url "https://github.com/mattn/go-runewidth.git",
-        :revision => "97311d9f7767e3d6f422ea06661bc2c7a19e8a5d"
-  end
-
-  go_resource "github.com/mitchellh/go-wordwrap" do
-    url "https://github.com/mitchellh/go-wordwrap.git",
-        :revision => "ad45545899c7b13c020ea92b2072220eefad42b8"
-  end
-
-  go_resource "github.com/ogier/pflag" do
-    url "https://github.com/ogier/pflag.git",
-        :revision => "45c278ab3607870051a2ea9040bb85fcb8557481"
-  end
-
-  go_resource "github.com/peterhellberg/link" do
-    url "https://github.com/peterhellberg/link.git",
-        :revision => "8768c6d4dc563b4a09f58ecda04997024452c057"
-  end
-
-  go_resource "github.com/satori/go.uuid" do
-    url "https://github.com/satori/go.uuid.git",
-        :revision => "36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
-  end
+  # Broken for new users since 2020-06-01
+  deprecate!
```

Comment:
maybe?
```suggestion
  deprecate! :date => "2020-06-01"
```

It would be nice to include some docs links into the code comment.